modules/debts_api/app/workers/debts_api/v0/form5655/vha/sharepoint_submission_job.rb

```diff
@@ -11,7 +11,8 @@ class V0::Form5655::VHA::SharepointSubmissionJob
     sidekiq_options retry: 4
 
     sidekiq_retries_exhausted do |job, _ex|
-      StatsD.increment("#{STATS_KEY}.failure")
+      StatsD.increment("#{STATS_KEY}.failure") # Deprecate this in favor of exhausted naming convention below
```

Comment:
Note we'll deprecate this once we have enough metrics collected using the new naming convention.